src/utils.js

```diff
@@ -17,13 +18,14 @@ function defaultOptions(endpointPath) {
   };
 }
 
+/**
+ * Properly joins paths together to create a URL.
+ */
 function makeUrl() {
```

Comment:
as in the other PR, I think this function doesn't make sense because `urljoin` is exactly the same

Reply:
It's not, urljoin failed the unit tests. Sometimes it would append two slashes together, for some reason doing it this way avoids that issue.